app/assets/javascripts/components/dropdown-menu.js

```diff
@@ -0,0 +1,31 @@
+ManageIQ.angular.app.component('dropdownMenu', {
+  bindings: {
+    id: '<',
+    buttonsData: '@',
+  },
+  controllerAs: 'vm',
+  controller: function() {
+    var vm = this;
+
+    this.$onInit = function() {
+      vm.dropdown_id = 'btn_' + vm.id;
+      vm.buttons = JSON.parse(vm.buttonsData);
+    };
+  },
+  template: [
```

Comment:
@ZitaNemeckova don't we want this in a static HTML?

Reply:
or HAML :wink:

Reply:
```haml
%div.dropdown.pull-right.dropdown-kebab-pf
  %button.btn.btn-link.dropdown-toggle{'aria-haspopup' => "true",
                                       'data-toggle'   => "dropdown",
                                       'id'            => "{{vm.dropdown_id}}",
                                       'type'          => "button",
                                       'aria-expanded' => "false"}
    %span.fa.fa-ellipsis-v

  %ul.dropdown-menu.dropdown-menu-right{'aria-labelledby' => "{{vm.dropdown_id}}"}
    %li{'ng-repeat' => "button in vm.buttons"}
      %a{'id'                => "{{button.id}}",
         'title'             => "{{button.title}}",
         'data-method'       => "{{button.dataMethod}}",
         'data-remote'       => "{{button.dataRemote}}",
         'confirm'           => "{{button.confirm}}",
         'href'              => "{{button.href}}",
         'data-miq_spark_on' => "{{button.sparkleOn}}"}
        %span{'ng-class' => "button.fonticon"}
          {{button.name}}
```